README.md

```diff
@@ -38,16 +38,17 @@ Two additional steps are needed to compile `.capnp` files to Go:
   GO111MODULE=off go get -u capnproto.org/go/capnp/v3/  # install go-capnproto to $GOPATH
   ```
 
-To learn how to compile a simple schema, [click here](https://github.com/capnproto/go-capnproto2/wiki/Getting-Started#compiling-schema-files).
+To learn how to compile a simple schema, [click here](docs/Getting-Started.md#compiling-schema-files).
```

Comment:
@lthibault, note that these section identifiers no longer exist after the changes you made; we should update those, I wanted to keep this pr smallish.